docs/docs/setup.md

```diff
@@ -34,12 +34,13 @@ adding a package to open database on the respective platform.
     dev_dependencies:
       drift_dev: ^{{ versions.drift_dev }}
       build_runner: ^{{ versions.build_runner }}
+      custom_lint: ^{{ versions.custom_lint }} 
```

Comment:
I think we should have a small section somewhere mentioning this explicitly (it can also be linked at the bottom of this page).

I'm wary about mentioning the `custom_lint` dependency here. We're already suggesting users install a lot of dependencies.
IMO, we should:

1. Mention somewhere that if users already have `custom_lint`, drift will work seamlessly with that and provide its own lints.
2. Suggest (but less strongly than done here, essentially pretending `custom_lint` is a requirement) that users try `custom_lint` to get quicker feedback about potential issues related to drift.

I think the best way to do that would be to mention this on [/Tools](https://drift.simonbinder.eu/Tools/) and then add a bullet point at the end of this page next to the others.